CONTRIBUTING.md

```diff
@@ -24,22 +28,27 @@ Or, on Unix you can run `./run-wheatley [ARGS]`.
 ├── wheatley
 │   ├── __init__.py                                 => Empty file
 │   ├── __main__.py                                 => Code to start Wheatley
-│   ├── arg_parsing.py                              => Code to parse formatted CLI args e.g. call strings
+│   ├── aliases.py
 │   ├── bell.py                                     => Stores the representation of a bell
 │   ├── bot.py                                      => The main class for Wheatley
 │   ├── calls.py                                    => Constants for the calls used in Ringing Room
 │   ├── main.py                                     => Entry point of Wheatley
 │   ├── page_parser.py                              => Code to parse the ringing room HTML
-│   ├── regression.py                               => Code for the weighted linear regression
-│   ├── rhythm.py                                   => Code used to drive Wheatley's rhythm
+│   ├── parsing.py                                  => Code to parse formatted CLI args e.g. call strings
+│   ├── stroke.py                                   => A module containing a type that encapsulates a stroke
 │   ├── tower.py                                    => Code used to interact with ringing room
+│   ├── rhythm.py                                   => Code used to drive Wheatley's rhythm
+|   |   ├── __init__.py
+|   |   ├── abstract_rhythm.py
+|   |   ├── regression.py                           => Code for the weighted linear regression
+|   |   ├── wait_for_user.py
```

Comment:
Nice! Thanks for updating this.  I'm planning to improve the documentation and this is a good start (if I'd known you were going to make a PR I'd have done it sooner, but you seem to have done well anyway).